hack/kiali/func-addons.sh

```
@@ -0,0 +1,220 @@
+  infomsg "Removing Addon: loki"
+  local addon_name="loki"
+  local yaml_file="/tmp/loki.yaml"
+  download_istio_addon_yaml "${addon_name}" "${yaml_file}"
+  delete_istio_addon_yaml "${yaml_file}"
+}
+
+# $1 = name of addon, $2 is the file path where the yaml is to be stored
+download_istio_addon_yaml() {
+  local addon_url="https://raw.githubusercontent.com/istio/istio/master/samples/addons/$1.yaml"
+  infomsg "Istio addon URL to download: $addon_url"
+  while ! curl --silent --output "$2" --location ${addon_url}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)